src/popup/components/layout/header.tsx

```diff
@@ -12,6 +12,14 @@ const NavigationButton = () => {
   return useMemo(() => {
     if (!connected) return null
 
+    if (location.pathname.includes('/s/')) {
+      return (
+        <Link to={location.pathname.split('/s/')[0]}>
+          <ArrowBack className="h-6" />
+        </Link>
+      )
+    }
+
```

Comment:
This solves https://github.com/interledger/web-monetization-extension/issues/424.

We can store `location` in localStorage, and use that as default route (`initialEntries`) in future, instead of managing screens from within components (at expense of using more pages).